flowmachine/tests/server/test_action_handlers.py

```diff
@@ -14,6 +17,21 @@
 from flowmachine.core.server.zmq_helpers import ZMQReplyStatus
 
 
+def test_run_query_type_error(monkeypatch):
```

Comment:
I've added this test because the `TypeError` code path wasn't touched, but I think we still want to keep it around for "the dev screwed up" situations, which is pretty much the only time it would occur.